sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/JoinSelectionHelperSuite.scala

```diff
@@ -103,15 +103,17 @@ class JoinSelectionHelperSuite extends PlanTest with JoinSelectionHelper {
   }
 
   test("getBroadcastBuildSide (hintOnly = false) return None when right has no broadcast hint") {
-    val broadcastSide = getBroadcastBuildSide(
-      left,
-      right,
-      Inner,
-      JoinHint(None, hintNotToBroadcast ),
-      hintOnly = false,
-      SQLConf.get
-    )
-    assert(broadcastSide === None)
+    withSQLConf(SQLConf.AUTO_BROADCASTJOIN_THRESHOLD.key -> "10MB") {
```

Comment:
Seems `10MB` is the default configuration value of `AUTO_BROADCASTJOIN_THRESHOLD`.
If the default value is change to `11MB`, does it need to be also changed here? Should `SQLConf.AUTO_BROADCASTJOIN_THRESHOLD.defaultValueString` be used?

Reply:
`getBroadcastBuildSide` looks at `AUTO_BROADCASTJOIN_THRESHOLD` (not default value) to make broadcast decision. I think this will not be affected if the default value is changed.

Reply:
@LuciferYang . This PR is cutting the test case assumption from the default value of `AUTO_BROADCASTJOIN_THRESHOLD`. After this PR, we can change the default value of `AUTO_BROADCASTJOIN_THRESHOLD` without any test failures. :)

Reply:
Yes, @viirya 's comment is correct.